samples/augment-reality-to-navigate-route/src/main/java/com/esri/arcgismaps/sample/augmentrealitytonavigateroute/components/SharedRepository.kt

```diff
@@ -0,0 +1,17 @@
+package com.esri.arcgismaps.sample.augmentrealitytonavigateroute.components
```

Comment:
missing copyright. There are other files that are also missing it. I will only comment here